source/guide_bitwarden.rst

```diff
@@ -0,0 +1,370 @@
+	nightly
+
+ Profile (which tools and data to install)? (minimal/default/complete)
+	default
+
+ Modify PATH variable? (y/n)
+	y
+
+ Current installation options:
+
+   default host triple: x86_64-unknown-linux-gnu
+     default toolchain: nightly
+               profile: default
+  modify PATH variable: yes
+
+ 1) Proceed with installation (default)
+ 2) Customize installation
+ 3) Cancel installation
+ >	1
+ [...]
+ Rust is installed now. Great!
+
+
+To finish the setup, **logout and login** again or run
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ source $HOME/.cargo/env
+
+Install Bitwarden_rs
+--------------------
+
+Clone the repository into your document root. It will create the directory ``~/bitwarden_rs`` automatically.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://github.com/dani-garcia/bitwarden_rs.git
+ 
+In order to build bitwarden_rs  sucessfully you'll need to set an environment variable pointing to the sqlite3 header files:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ export SQLITE3_LIB_DIR=/var/lib64
+
+``cd`` into the automatically created folder:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd bitwarden_rs
+ [isabell@stardust bitwarden_rs]$ 
+
+Build the server executable:
+
+.. note :: If that doesn't work the first time and the build failed, **just try again**.
+
+.. code-block:: console
+
+ [isabell@stardust bitwarden_rs]$ cargo build --release --features sqlite
+
+In the next step we will download the latest build for the web vault. Check `this page`_ for the newest build number and **replace it** in the following snippet:
+
+.. note :: If you don't want to use the web-vault feature (web-app access to your vaults) for any security reasons you can skip this step. Add the following line to your ``env`` file later if you do so:
+  ``WEB_VAULT_ENABLED=false``
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust bitwarden_rs]$ [isabell@stardust bitwarden_rs]$ wget https://github.com/dani-garcia/bw_web_builds/releases/download/v2.13.2/bw_web_v2.13.2.tar.gz
+ [isabell@stardust bitwarden_rs]$ tar -xvzf bw_web_v2.13.2.tar.gz
+ [isabell@stardust bitwarden_rs]$ rm -r bw_web_v*.tar.gz
+
+ 
+Generate an ``openssl -base64`` key now and save it temporarily, you'll need it in the next step.
+
+.. code-block:: console
+
+ [isabell@stardust bitwarden_rs]$  openssl rand -base64 48
+ [isabell@stardust bitwarden_rs]$  Ig/8bZXhqVFK11F2tQZTfODO/6QpCHE3DGyCH/2Eh40xUWMFC13J6nJVPLlyU3nO
+
+Use your favourite editor to create ``~/bitwarden_rs/.env`` with the following content:
+
+.. code-block:: ini
+
+ ADMIN_TOKEN=PASTE YOUR TOKEN GENERATED ABOVE HERE # generate one with ~$ openssl rand -base64 48
+ ROCKET_PORT=62714 # your port here
+
+ SMTP_HOST=stardust.uberspace.de
+ SMTP_FROM=isabell@uber.space
+ SMTP_PORT=587
+ SMTP_SSL=true
+ SMTP_USERNAME=isabell@uber.space
+ SMTP_PASSWORD=MySuperSecretPassword
+ 
+ DOMAIN=https://isabell.uber.space:8443
```

Comment:
the port on the end is wrong (email verification do not work)

Reply:
You're right! Thanks, i removed the port.